lib/saturn-node.js

```diff
@@ -81,16 +69,15 @@ async function start ({
       if (apiMatch) {
         const apiUrl = apiMatch[1]
 
-        logStream.write('Saturn node is up and ready')
         childProcess.stdout.off('data', readyHandler)
-        activityStream.write({
+        activities.emit('activity', {
```

Comment:
❗ 

I think you need to add a `source` field to tell which module is the source of this activity. 

**This comment applies to all other places in this file where you are calling `activities.emit`, and all other modules (Bacalhau, Zinnia) too.**

My suggestion: create a helper function replacing `activityStream.write` and setting the activity source.

```
// at the top start() function body
const logActivity = ({ type, message }) => {
  activities.emit('activity', { source: 'Saturn', type, message });
}

// here on line 73
logActivity({
  type: 'info',
  message: 'Saturn module started.'
})
```